tests/js/client/resilience/foxxmaster.js

```diff
@@ -96,7 +96,6 @@ function FoxxmasterSuite() {
   return {
     setUp: function() {
       serverSetup();
-      wait(2.1);
```

Comment:
I'm unsure if this is needed or not

Reply:
@graetzer @mchacki

Reply:
hm seems like a very specific timeout. Maybe to allow for self-healt to install the foxx app everywehere